index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div class="form-title">
+      <h1 id="title">Survey Form</h1>
+      <p id="description">This is our first survey form project, please fill out.</p>
+    </div>
```

Comment:
Hi @gabyse1, let's make use of the semantic `<header>` tag. Always make use of the semantic tags where necessary, it provides additional information about different sections and layout of webpages.